newIDE/app/src/UI/EmptyPlaceholder.js

```diff
@@ -48,22 +52,36 @@ export const EmptyPlaceholder = (props: Props) => (
         </Text>
         <LargeSpacer />
         <ColumnStackLayout alignItems="center" noMargin>
-          <RaisedButton
-            label={props.actionLabel}
-            primary
-            onClick={props.onAction}
-            disabled={!!props.isLoading}
-            icon={
-              props.isLoading ? (
-                <CircularProgress size={24} />
-              ) : props.actionIcon ? (
-                props.actionIcon
-              ) : (
-                <Add />
-              )
-            }
-            id={props.actionButtonId}
-          />
+          <Line noMargin>
+            {props.secondaryActionLabel && (
```

Comment:
To be cleaner, you should check the existence of the other props that is kind of mandatory: onSecondaryAction